screens/home/HomeFeed.tsx

```diff
@@ -18,44 +20,63 @@ const HomeFeed = () => {
     process.env.NODE_ENV !== 'development'
   );
   const [cursor, setCursor] = useState<PostCursorMerger>();
+  const [exhausted, setExhausted] = useState<boolean>(false);
   const [posts, setPosts] = useState<PostObj[]>([]);
 
   useEffect(() => {
     if (enabled && signedInUser !== undefined && userQuery.data !== undefined) {
-      console.log("Ok let's get it goin!");
       const newCursor = getQueryCursor(userQuery.data.followingList);
```

Comment:
Will this only show Posts from people the user is following? There should also be an option to see posts from everyone, even if you don't follow them. Or will that be implemented in a different PR?

Reply:
This is currently only posts for whom  you're following. I didn't know we were doing posts from everyone; that can be a different PR